core/src/main/java/com/gdx/game/quest/QuestGraph.java

```diff
@@ -28,6 +25,11 @@ public class QuestGraph {
     private int goldReward;
     private int xpReward;
 
+    public QuestGraph() {
```

Comment:
QuestGraph should not be initialized with a constructor (except for test)
If you want to initialize some variables, do it inside the class itself